code/__HELPERS/multiz_helpers.dm

```diff
@@ -108,8 +108,6 @@ GLOBAL_VAR_TYPED(temporary_multiz_step_ref, /turf)
 	if (above)
 		above.z_depth = null
 		above.below = null
-		// Cleanup z-mimic stuff as it is no longer being used
-		above.cleanup_zmimic()
```

Comment:
This would result in the old turf holding a mimic of the previous below turf, would it not?